interface/patient_file/encounter/forms.php

```diff
@@ -21,16 +21,26 @@
 <html>
 
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<?php html_header_show();
+//  Include Bootstrap Library
+call_required_libraries(true,false,false,false);
+?>
 <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['standard_js_path']?>fancybox-1.3.4/jquery.fancybox-1.3.4.css" media="screen" />
 <style type="text/css">@import url(../../../library/dynarch_calendar.css);</style>
```

Comment:
this should not be needed with the new date picker. I must have missed removing it.